### PR TITLE
adds logic for  large mixer multi

### PR DESCRIPTION
**Pull request by last8kings**

### CODE REVIEW

File: src/main/java/supersymmetry/api/recipes/SuSyRecipeMaps.java [hunk 1]
```diff
@@ -77,7 +80,6 @@ public class SuSyRecipeMaps {
             .setSound(GTSoundEvents.CHEMICAL_REACTOR);
 
     public static final RecipeMap<SimpleRecipeBuilder> DRYER = new RecipeMap<>("dryer", 1, 2, 1, 1, new SimpleRecipeBuilder(), false)
-            .setSound(GTSoundEvents.COOLING);
```
Comment (planetme) [inline src/main/java/supersymmetry/api/recipes/SuSyRecipeMaps.java:80]: ?

File: src/main/java/supersymmetry/api/recipes/SuSyRecipeMaps.java [hunk 2]
```diff
@@ -108,7 +110,7 @@ public class SuSyRecipeMaps {
             .setSlotOverlay(true, false, SusyGuiTextures.LARGE_REACTOR_ITEM_OVERLAY)
             .setSound(GTSoundEvents.CHEMICAL_REACTOR);
 
-    public static final RecipeMap<CatalystRecipeBuilder> POLYMERIZATION_RECIPES = new RecipeMap<>("polymerization_tank", 2, 1, 3, 1, new CatalystRecipeBuilder(), false)
+    public static final RecipeMap<CatalystRecipeBuilder> POLYMERIZATION_RECIPES = new RecipeMap<>("polymerization_tank", 3, 1, 3, 2, new CatalystRecipeBuilder(), false)
```
Comment (planetme) [inline src/main/java/supersymmetry/api/recipes/SuSyRecipeMaps.java:113]: what did you base this branch on?

File: src/main/java/supersymmetry/api/recipes/SuSyRecipeMaps.java [hunk 3]
```diff
@@ -301,6 +303,9 @@ public class SuSyRecipeMaps {
     public static final RecipeMap<SimpleRecipeBuilder> POLISHING_MACHINE = new RecipeMap<>("polishing_machine", 1, 1, 2, 1, new SimpleRecipeBuilder(), false)
             .setProgressBar(GuiTextures.PROGRESS_BAR_MIXER, ProgressWidget.MoveType.CIRCULAR)
             .setSound(GTSoundEvents.CENTRIFUGE);
+    public static final RecipeMap<SimpleRecipeBuilder> LARGE_MIXER_RECIPES = new RecipeMap<>("large_mixer", 7, 7, 5, 4, new SimpleRecipeBuilder().EUt(VA[LV]), false)
```
Comment (planetme) [inline src/main/java/supersymmetry/api/recipes/SuSyRecipeMaps.java:306]: no need for so many outputs, 1 fluid and item output

**Review by planetme — CHANGES_REQUESTED** [review]

File: src/main/java/supersymmetry/api/recipes/SuSyRecipeMaps.java (hunk 2 above)
Comment (last8kings) [reply src/main/java/supersymmetry/api/recipes/SuSyRecipeMaps.java:113]: i based it off main then cherry picked the commit to a new branch and then realized I made a mistake and tried going back and undoing all the new changes

File: src/main/java/supersymmetry/api/recipes/SuSyRecipeMaps.java (hunk 3 above)
Comment (last8kings) [reply src/main/java/supersymmetry/api/recipes/SuSyRecipeMaps.java:306]: ok

**Comment by last8kings** [comment]
ok wait im gonna reopen the PR and make a branch properly this time

**Pull request closed without merging**